src/main/java/org/omnifaces/util/Utils.java

```diff
@@ -776,8 +763,11 @@ public static <T> Stream<T> stream(Object object) {
         if (object == null) {
             return Stream.empty();
         }
+        if (object instanceof Collection) {
+            return ((Collection<T>)object).stream();
```

Comment:
Try to be consistent with code format. Lack of blank space after cast operator deviates from existing code.